Makefile

```diff
@@ -63,3 +65,11 @@ $(DEPS_INCLUDEDIR)/plugin_api.h: $(DEPS_INCLUDEDIR)
 
 $(DEPS_INCLUDEDIR)/nlohmann/json.hpp: $(DEPS_INCLUDEDIR)
 	+$(CURL) -sLo $(DEPS_INCLUDEDIR)/nlohmann/json.hpp https://github.com/nlohmann/json/releases/download/v3.10.2/json.hpp
+
+.PHONY: install
+install: 
+	cp -r $(INCLUDE_DIR) $(INSTALL_DIR)
+
+.PHONY: uninstall
+uninstall:
```

Comment:
Will this completely erase /usr/include ? 😲

Reply:
:exploding_head:

Reply:
Yes sorry about that ahah fixing

Reply:
:rofl: